test/libprojecttest.cpp

```diff
@@ -33,370 +33,185 @@ list<FileSetLoader::Path> pathsToSingleAbnormal = {
-
-          { "files", { "main.cpp", "Test.h" } },
-
-          { "subprojects", {"presentsubproject/sub.libproject",
-                            "regular/normalsingle.libproject"} }
-        };
-        contentReferenceWithNewNested = {
-          { "project", "there must be subprojects" },
-
-          { "files", { "main.cpp", "Test.h" } },
-
-          { "subprojects", {"presentsubproject/sub.libproject",
-                            "regular/normalnested.libproject"} }
-        };
-        contentReference = {
-          { "project", "there must be subprojects" },
-
-          { "files", { "main.cpp", "Test.h" } },
-
-          { "subprojects", {"presentsubproject/sub.libproject"} }
-        };
-        contentReferenceWithPairOfNewRegularSubprojects = {
-            { "project", "there must be subprojects" },
-
-            { "files", { "main.cpp", "Test.h" } },
-
-            { "subprojects", {"presentsubproject/sub.libproject",
-                              "fortestwithtworegularsubprojects/nested.libproject",
-                              "fortestwithtworegularsubprojects/single.libproject"} }
-        };
-        pathToOneRegularSingleSubproject =
-            R"(project_files/testaddtonested/regular/normalsingle.libproject)";
-
-        pathToOneRegularNestedSubproject =
-            R"(project_files/testaddtonested/regular/normalnested.libproject)";
-
-        pathToBrokenSubproject = {
-            "project_files/testaddtonested/broken/broken.libproject"
-        };
-        pathsToPairOfRegularSubprojects = {
-            "project_files/testaddtonested/fortestwithtworegularsubprojects/single.libproject",
-            "project_files/testaddtonested/fortestwithtworegularsubprojects/nested.libproject"
-        };
-        pathsToPairOfRegularAndBrokenSubprojects = {
-            "project_files/testaddtonested/fortestwithonebrokenandoneregularsubprojects/normal.libproject",
-            "project_files/testaddtonested/fortestwithonebrokenandoneregularsubprojects/broken.libproject"
-        };
-        pathsToPairOfBrokenSubprojects = {
-            "project_files/testaddtonested/fortestwithtwobrokensubprojects/one.libproject",
-            "project_files/testaddtonested/fortestwithtwobrokensubprojects/two.libproject"
-        };
-        pathsToPairOfRegularAndEmptySubprojects = {
-            "project_files/testaddtonested/fortestwithoneregularandemptysubprojects/regular.libproject",
-            "project_files/testaddtonested/fortestwithoneregularandemptysubprojects/empty.libproject"
-        };
-
-        ifstream in(pathToMainFile);
-        ostringstream os;
-        char buf;
-        while(in.get(buf))
-          os << buf;
-        in.close();
-        contentBackup = os.str();
-
         TestSkeleton::SetUp();
     }
-
     void TearDown() {
-      ofstream out (pathToMainFile);
-      for (const auto& sym : contentBackup)
-        out << sym;
-      out.close();
+        TestSkeleton::TearDown();
```

Comment:
@mkwork 
In TestSkeleton::TearDown and SetUp I put things that performs some work. Why "do nothing"?